frontend/configuration.properties.js

```diff
@@ -448,4 +455,4 @@
     // only show links to documents that come from tables other than the one the main document is in.
     includeAllTables: false,
   },
-}
+}
```

Comment:
Linting issue: Add new line at EOF
```suggestion
}

```